docs/source/models/intervention_models/neonatal/antibiotics_intervention.rst

```diff
@@ -95,12 +154,32 @@ The following decision tree shows all of the paths from delivery facility choice
         p(\text{sepsis}) 
         &= \sum_{\text{paths without antibiotics}} p(\text{path})\cdot p(\text{sepsis}|\text{no antibiotics})\\
         &+ \sum_{\text{paths with antibiotics}} p(\text{path})\cdot p(\text{sepsis}|\text{antibiotics})\\[.1in]
-        p(\text{sepsis}|\text{antibiotics}) &= \text{RR}_\text{antibiotics} \cdot p(\text{sepsis}|\text{no antibiotics})
+        p(\text{sepsis}|\text{no antibiotics}) &= \text{RR}_\text{no antibiotics} \cdot p(\text{sepsis}|\text{antibiotics})
     \end{align*}
 
 where :math:`p(\text{sepsis})` is the probability of dying from sepsis in the general population, and :math:`p(\text{sepsis}|\text{antibiotics})` and :math:`p(\text{sepsis}|\text{no antibiotics})` are the probability of dying from sepsis in setting with and without access to antibiotics.  For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has antibiotics available.
 
-When we fill in the location-specific values for delivery facility rates, antibiotics coverage, relative risk of mortality with antibiotics access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_sepsis_w and p_sepsis_wo), which we can solve analytically.
+When we fill in the location-specific values for delivery facility rates, antibiotics coverage, relative risk of mortality with antibiotics access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_sepsis_w and p_sepsis_wo), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.
+
+**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that :math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
+
+.. math::
+
+   p(\text{no antibiotics}) = \sum_{\text{paths without antibiotics}} p(\text{path}),
+
+then can use this to expand the identity
+
+.. math::
+
+   \text{PAF}_\text{no antibiotics} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.
+
+Since our risk exposure has two categories,
+
+.. math::
+
+   \mathbb{E}(\text{RR}) = p(\text{no antibiotics}) \cdot \text{RR}_\text{no antibiotics} + (1 - p(\text{no antibiotics})) \cdot 1.
```

Comment:
Is there a reason not to make this the primary explanation on both pages? The way this is written, it seems to imply that this is an option for antibiotics but *not* for CPAP, but it seems like it would apply equally to either.

Reply:
Jim already implementing things for CPAP, so I thought I'd leave it the way it was when he implemented, at least for now.

Reply:
Makes sense. Can we add this alternative to both pages then, or explicitly note that it can work for either? Like I said I was a bit thrown off by its placement/its absence on the other page.

Reply:
I'm fine with a explicit note and a link.  Do you have time to add it?